src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -166,7 +171,12 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     embedImages: EmbedImages,
     embedUploader: Uploader,
     group: gist,
-    links: (props: ComponentToRenderProps) => <ScrollablePromo {...props} />,
+    links: (props: ComponentToRenderProps) =>
+      scrollablePromoVariation === 'variation_1_aa' ? (
+        <ScrollablePromo {...props} />
+      ) : (
+        <ScrollablePromo {...props} />
+      ),
```

Comment:
I don't know a huge amount about optimizely, but I assume that if we had an AB test, then we would do something different for 1 variant. What if we did nothing here (i.e. revert the changes to this line), would it still work? I suppose we'll never really do AA tests - this is just to prove that we get the expected results in Optimizely, right?

Reply:
Yeah this is just to prove that we get metrics through for both variations we've set up. I also dont know much about optimizely lol but assume yea if this was an AB test we'd render a different component depending on the variant. Good Q about doing nothing there, i think yes it would still work as the variant is calculated by optimizely based off of a userId set with a cookie (will defer to @HarveyPeachey to fact check me there though), but good practice to leave it there for clarity and testing's sake

Reply:
Could/should we do something like wrap the A test variant in a div with the variation ID, or is that overkill?

e.g. 

```suggestion
    links: (props: ComponentToRenderProps) =>
      scrollablePromoVariation === 'variation_1_aa' ? (
        <div data-testid={scrollablePromoVariation}>
           <ScrollablePromo {...props} />
        </div>
      ) : (
        <ScrollablePromo {...props} />
      ),
```

Reply:
would this be so we can see it coming through in dev tools when we visit it ? i think that could be overkill,  we'll be tracking what variants come through to users through optimizely, so dont think we need to be able to see the variant id ourselves - if it's not working we'll be able to tell bc no metrics will come through

Reply:
Yeah thought we could add a unit test for it, but perhaps it is overkill. I've approved anyway, so feel free to merge when it's ready to go!

Reply:
Yeah AA tests are only for testing this, as Emily said, if it was an a proper experiment we would see something else render based on variation. And yeah this is just to prove everything is working